.gitignore

```diff
@@ -3,6 +3,10 @@ cmd/fyne/fyne
 cmd/fyne_demo/fyne_demo
 cmd/fyne_settings/fyne_settings
 cmd/hello/hello
+cmd/fyne/fyne.exe
+cmd/fyne_demo/fyne_demo.exe
+cmd/fyne_settings/fyne_settings.exe
+cmd/hello/hello.exe
```

Comment:
This change is unrelated

Reply:
Do you want a separate issue raised for the .gitignore?

Reply:
Looks to me like it was cleanly added in a separate commit, I am sure that we can merge the PR with this in it.
The overhead for a PR process is sometimes just not worth it for single commits like this metadata.
As far as I am concerned as long as each commit has clear purpose and they are not distinct discussion points for unrelated work then it's not a problem.

Reply:
ack